.ci/docker/common/install_base.sh

```diff
@@ -34,6 +34,8 @@ install_ubuntu() {
   # See https://github.com/pytorch/pytorch/pull/105260#issuecomment-1673399729
   if [[ "$UBUNTU_VERSION" == "20.04"* && "$CUDA_VERSION" == "11.8"* ]]; then
     maybe_libnccl_dev="libnccl2=2.15.5-1+cuda11.8 libnccl-dev=2.15.5-1+cuda11.8 --allow-downgrades --allow-change-held-packages"
```

Comment:
not related to this PR but curious why for CUDA 11.8 we use NCCL 2.15 instead of 2.21?